src/components/panel/Panel.js

```diff
@@ -33,20 +33,20 @@ export default class PanelComponent extends NestedComponent {
   get defaultSchema() {
     return PanelComponent.schema();
   }
-
-  checkValidity(data, dirty, row, silentCheck) {
-    if (!this.checkCondition(row, data)) {
-      this.setCustomValidity('');
-      return true;
-    }
-
-    return this.getComponents().reduce(
-      (check, comp) => {
-        return comp.checkValidity(data, dirty, row, silentCheck) && check;
-      },
-      super.checkValidity(data, dirty, row, silentCheck)
-    );
-  }
+  //
```

Comment:
I would rather just delete this code instead of comment it out. We can use GIT to see what was removed.